src/modals/masq_activating_modal.js

```diff
@@ -0,0 +1,44 @@
+import MasqActivatingModalView from '../views/masq_activating_modal.dot'
+import Modal from "./modal"
+
+export default class MasqActivatingModal {
+  constructor() {
+    // get masqErrorModal from window if already initialized
+    if (window.__masqActivatingModal) {
+      return window.__masqActivatingModal
+    }
+    // if masqErrorModal not initialized, use this
+    window.__masqActivatingModal = this
+
+    this.modal = new Modal(this, MasqActivatingModalView, false)
+
+    this.opened = false
```

Comment:
this flag is not used. And as `this.activating` is true by default, the modal and the video are present in DOM at app start. (Chromium at least tries to load the .mp4 before it is visible)

Reply:
Fixed !
removed opened flag and made activating false by default